WorkingwithAnsibleAdHocCommands.md

```diff
@@ -0,0 +1,210 @@
+## Working with Ansible Ad-Hoc Commands on EXOS Devices
+
+## Sections:
+
+* [Introduction](#introduction)
+* [Pre-requisite](#pre-requisite)
+* [Basic Ansible Commands](#basic-ansible-commands)
+	- [Check Ansible Version](#check-ansible-version)
+	- [Check available hosts in Ansible Inventory](#check-available-hosts-in-ansible-inventory)
+	- [Ansible Documentation](#ansible-documentation)
+	- [Ansible Help](#ansible-help)
+* [Basic Ansible Ad-Hoc Commands](#basic-ansible-ad-hoc-commands)
+	- [Pinging Hosts](#pinging-hosts)
+	- [Gathering Facts](#gathering-facts)
+	- [Get Version Details](#get-version-details)
+	- [Passing Multiple show Commands](#passing-multiple-show-commands)
+  	- [Logging in with a different user](#logging-in-with-a-different-user)
+  	- [Filtering Output](#filtering-output)
+  	- [Saving Output in a File](#saving-output-in-a-file)
+* [Conclusion](#conclusion)
+
+## Introduction
+
+An ad-hoc command is something that you might type in order to do something really quick, but don’t want to save for later.
+
+This is a good place to start to understand the basics of what Ansible can do prior to learning the playbooks language. Ad-hoc commands can also be used to do quick things that you might not necessarily want to write a full playbook for.
+
+This guide will be focused mainly on the application of these commands on Network devices, and in our case its for EXOS devices.
+
+## Pre-requisite:
+
+A working setup of Ansible system with at least one EXOS device.
+
+> Refer [this](https://github.com/extremenetworks/ansible-extreme/blob/master/GettingStartedwithAnsibleusingGNS3.md) guide to prepare a similar setup on GNS3
```

Comment:
Probably better to make this link relative (i.e. something like `./GettingStartedwithAnsibleusingGNS3.md`, rather than absolute (`https://...`). That way it still works if the repo is renamed or forked

Reply:
Its done